_posts/2019-07-16-ion-hash-python-1_0_1-released.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: news_item
+title: "Ion Hash Python 1.0.1 Released"
+date: 2019-07-16
+categories: news
+---
+This release is a complete implementation of the [Ion Hash Specification](https://amzn.github.io/ion-hash/docs/spec.html). In addition to low-level `hash_reader`/`hash_writer` APIs, it adds an `ion_hash()` method to all simpleion objects.

```

Comment:
Maybe add a link to simpleion so it's clear that that refers to ion-python.

Reply:
added a link to ion-python where readers can learn more about simpleion; let me know if you had something else in mind